### PR TITLE
FAQ - update 'qr_test' entry

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- Link to FAQ entry with TAN Hotlines instead of mentioning phone numbers directly
- Add missing single quote: 'Display Test' -> 'Delete test

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -188,12 +188,11 @@
                             "<ul><li>the respective laboratory is not fully linked with the Corona Warn App backend.</li><li>the consent field allowing the transmission of the test result to the Corona-Warn-App was not ticked on the submission form (10C/OEGD) sent by your doctor to the laboratory.</li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq.json:188]: "Corona Warn App" => "Corona-Warn-App" like in the following line for naming consistency.

File: src/data/faq.json [hunk 2]
```diff
@@ -188,12 +188,11 @@
...
-                            "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a TeleTAN by calling the TAN hotline on 0800 7540002 to warn your contacts via the Corona-Warn-App. Before you are able to enter the TeleTAN into the app, you have to delete the pending test result from the app. To do this, please tap on",
-                            "<ul><li>'Display Test' -> 'Delete test</li></ul>",
+                            "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a TeleTAN from the TAN Hotline (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>click here to see our phone numbers</a>) to warn your contacts via the Corona-Warn-App. Before you are able to enter the TeleTAN into the app, you have to delete the pending test result from the app. To do this, please tap on",
```
Comment (MikeMcC399) [inline src/data/faq.json:191]: The link to #international_phone_numbers only gives the TAN hotline +49 30 498 75402 which is not free-of-charge like the 0800 7540002 number which was previously listed.

Comment (Ein-Tim) [reply src/data/faq.json:191]: I had the same thought but there's a PR for this: #1078

File: src/data/faq_de.json [hunk 3]
```diff
@@ -187,13 +187,12 @@
...
-                            "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die TAN-Hotline unter der 0800 7540002 eine TeleTAN beantragen, um über die Corona-Warn-App Ihre Kontakte zu warnen. Bevor Sie die TeleTAN in die App eintragen können, müssen Sie erst das noch ausstehende Testergebnis aus der App löschen. Dazu tippen Sie bitte auf ",
+                            "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die TAN-Hotline eine TeleTAN beantragen (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>), um über die Corona-Warn-App Ihre Kontakte zu warnen. Bevor Sie die TeleTAN in die App eintragen können, müssen Sie erst das noch ausstehende Testergebnis aus der App löschen. Dazu tippen Sie bitte auf ",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:190]: Same comment here, that the link does not include the free-of-charge number.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
One change of app name from "Corona Warn App" => "Corona-Warn-App" for consistency.
In addition I commented that the link to the international phone numbers doesn't include the 0800 free-of-charge number.

**Comment by MikeMcC399** [comment]
This PR should not be merged until https://github.com/corona-warn-app/cwa-website/pull/1078 is merged.

File: src/data/faq.json (hunk 2 above)
Comment (heinezen) [reply src/data/faq.json:191]: yes, they will all link to a general hotline article with national/international numbers

File: src/data/faq.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq.json]: fixed

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
@heinezen @MikeMcC399 @Ein-Tim  Thanks.

**Pull request merged**